CppSamples/DisplayInformation/ShowGrid/main.cpp

```diff
@@ -0,0 +1,96 @@
+// Copyright 2024 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "ShowGrid.h"
+
+#include "ArcGISRuntimeEnvironment.h"
+
+#include <QCommandLineParser>
+#include <QDir>
+#include <QGuiApplication>
+#include <QQmlApplicationEngine>
+#include <QSurfaceFormat>
+
+#ifdef Q_OS_WIN
+#include <Windows.h>
+#endif
+
+void setAPIKey(const QGuiApplication& app, QString apiKey);
+
+int main(int argc, char *argv[])
+{
+#if defined(Q_OS_LINUX) && !defined(Q_OS_ANDROID)
+  // Linux requires 3.2 OpenGL Context
+  // in order to instance 3D symbols
+  QSurfaceFormat fmt = QSurfaceFormat::defaultFormat();
+  fmt.setVersion(3, 2);
+  QSurfaceFormat::setDefaultFormat(fmt);
+#endif
+
+  QGuiApplication app(argc, argv);
+  app.setApplicationName(QString("ShowGrid - C++"));
+
+  // Access to Esri location services requires an API key. This can be copied below or used as a command line argument.
+  const QString apiKey = QString("");
+  setAPIKey(app, apiKey);
```

Comment:
❌ this API Key terminology all looks old. The samples and templates should've been updated to use the "access token" terminology